cmd/root.go

```diff
@@ -84,7 +99,11 @@ func NewPolycliCommand() *cobra.Command {
 		Short: "A Swiss Army knife of blockchain tools.",
 		Long:  "Polycli is a collection of tools that are meant to be useful while building, testing, and running block chain applications.",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			setLogLevel(verbosity, pretty)
+			if cmd.Use == monitor.MonitorCmd.Use {
```

Comment:
I think this may get messy if we want to override a lot in the future. Have you tried overriding the `PersistentPreRun` method in the monitor command? I'm not sure if it'll work.

Reply:
100% agree with this, I tried to use `inherited` flags but it seems to be a dead end.

Reply:
Way better like this: [86de636](https://github.com/maticnetwork/polygon-cli/pull/154/commits/86de6361271edf7f4a14424bc188820ba82b2ecb)

Reply:
If you try to override the verbosity, will it still work?

> Way better like this: [86de636](https://github.com/maticnetwork/polygon-cli/pull/154/commits/86de6361271edf7f4a14424bc188820ba82b2ecb)

Reply:
Yup doesn't work lol, I reverted the commit :)